### PR TITLE
JavaScript Workshop 1 Complete

### PR DESCRIPTION
**Pull request by JordanGabay**
Struggled with question 9... Was not able to put FizzBuzz without it covering all sums of 3 - 100.
Was able to get everything else to work!

### CODE REVIEW

File: workshop/exercise-9.js [hunk 1]
```diff
@@ -24,3 +24,15 @@
 // FizzBuzz
 // 16
 // ...
+
+
+for (num = 0; num < 100; num++) {
```
Comment (SSaquif) [inline workshop/exercise-9.js:29]: This solution is incorrect (so is the solution in the solutions folder), as it would never print out fizzbuzz

Comment (JordanGabay) [reply workshop/exercise-9.js:29]: Hey! Sometimes I have to rely on the solution to see the steps more then I would like to but I'm always trying to find other ways to do it, mostly working with my team. Through the JavaScript modules I definitely have gotten a feeling to get on the queue more so I'll be sure to do that.

Comment (SSaquif) [reply workshop/exercise-9.js:29]: I understand, didn't mean to be negative. Just to let you know, we are here to help and stop u from getting into a bad habit early on. But, looking at the solutions is completely fine to get a better understanding. Also sometimes the solutons are not perhaps up to date with the changes made in the workshops by scott.

Comment (JordanGabay) [reply workshop/exercise-9.js:29]: Oh dont worry! I didn't think you were being negative at all haha. I completely understand that not all the solutions are up to date and I appreciate all the help you guys are offering! It's really made the difference.

**Pull request closed without merging**